client/src/models/covid/queries.js

```diff
@@ -3,286 +3,230 @@ import _ from "lodash";
 
 import { lang } from "src/core/injected_build_constants.ts";
 
-import {
-  get_client,
-  query_logging_wrapper,
-} from "src/graphql_utils/graphql_utils.js";
-
-const client = get_client();
+import { query_maker } from "src/graphql_utils/graphql_utils.js";
 
 const years_with_covid_data = `
   years_with_covid_data {
     years_with_estimates
     years_with_expenditures
   }
 `;
-export const query_gov_years_with_covid_data = query_logging_wrapper(
-  "gov_years_with_covid_data",
-  ({ ...logging_variables }) =>
-    client
-      .query({
-        query: gql`
-          query($lang: String!) {
-            root(lang: $lang) {
-              gov {
-                id
-                ${years_with_covid_data}
-              }
-            }
-          }
-        `,
-        variables: {
-          lang,
-          ...logging_variables,
-        },
-      })
-      .then((response) =>
-        _.get(response, "data.root.gov.years_with_covid_data")
-      )
-);
-export const query_org_years_with_covid_data = query_logging_wrapper(
-  "org_years_with_covid_data",
-  ({ org_id, ...logging_variables }) =>
-    client
-      .query({
-        query: gql`
-          query($lang: String!, $org_id: String!) {
-            root(lang: $lang) {
-              org(org_id: $org_id) {
-                id
-                ${years_with_covid_data}
-              }
-            }
-          }
-        `,
-        variables: {
-          lang,
-          org_id,
-          ...logging_variables,
-        },
-      })
-      .then((response) =>
-        _.get(response, "data.root.org.years_with_covid_data")
-      )
-);
+export const {
+  query_gov_years_with_covid_data,
+  useGovYearsWithCovidData,
+} = query_maker({
+  query_name: "gov_years_with_covid_data",
+  query: gql`
+    query($lang: String! = "${lang}") {
+      root(lang: $lang) {
+        gov {
+          id
+          ${years_with_covid_data}
+        }
+      }
+    }
+  `,
+  resolver: (response) =>
+    _.get(response, "data.root.gov.years_with_covid_data"),
+});
+export const {
+  query_org_years_with_covid_data,
+  useOrgYearsWithCovidData,
+} = query_maker({
+  query_name: "org_years_with_covid_data",
+  query: gql`
+    query($lang: String! = "${lang}", $org_id: String!) {
```

Comment:
For inline query variable defaults, like `$top_x` below, I think the query definition is definitely the place to put them. Actually a bit conflicted about adding the default `lang` value inline though. Most likely going to lift passing a default lang value back up in to the `query_maker`.